win32ss/user/ntuser/sysparams.c

```diff
@@ -301,7 +301,7 @@ SpiUpdatePerUserSystemParameters(VOID)
     gspv.bBeep = TRUE;
     gspv.uiFocusBorderWidth = 1;
     gspv.uiFocusBorderHeight = 1;
-    gspv.bMenuDropAlign = 1;
+    gspv.bMenuDropAlign = 0;
```

Comment:
Why?

Reply:
![VirtualBox_Windows 2003_08_12_2019_12_10_47](https://user-images.githubusercontent.com/2107452/70383603-cef19280-19b3-11ea-9e1e-1d5577f35380.png)
There is no such a registry value in Win2k3.

Reply:
OK.